time="2025-04-13T09:08:39.861535067Z" level=info msg="Starting Dapr Runtime -- version 1.15.4 -- commit eb94584f0e14b9fb61d2d139277c2962c1567064" app_id=python-subscriber instance=python-subscriber-bc75cbf4d-cbbks scope=dapr.runtime type=log ver=1.15.4
time="2025-04-13T09:08:39.861579169Z" level=info msg="Log level set to: debug" app_id=python-subscriber instance=python-subscriber-bc75cbf4d-cbbks scope=dapr.runtime type=log ver=1.15.4
time="2025-04-13T09:08:39.862080595Z" level=info msg="Fetching initial identity certificate" app_id=python-subscriber instance=python-subscriber-bc75cbf4d-cbbks scope=dapr.runtime.security type=log ver=1.15.4
time="2025-04-13T09:08:57.631051343Z" level=info msg="Received signal 'terminated'; beginning shutdown" app_id=python-subscriber instance=python-subscriber-bc75cbf4d-cbbks scope=dapr.signals type=log ver=1.15.4
time="2025-04-13T09:08:57.631309856Z" level=fatal msg="Fatal error from runtime: failed to retrieve the initial identity certificate: error establishing connection to sentry: context canceled: connection error: desc = \"transport: authentication handshake failed: x509svid: could not verify leaf certificate: x509: certificate signed by unknown authority (possibly because of \\\"x509: ECDSA verification failure\\\" while trying to verify candidate authority certificate \\\"cluster.local\\\")\"" app_id=python-subscriber instance=python-subscriber-bc75cbf4d-cbbks scope=dapr.runtime type=log ver=1.15.4
